core/toolbox.js

```diff
@@ -252,6 +252,163 @@ Blockly.Toolbox.prototype.renderTree = function(languageTree) {
   }
 };
 
+/**
+ * Create the toolbox tree.
+ * @param {Array.<Blockly.utils.toolbox.ToolboxInfo>} toolboxDef List of objects
+ *    holding information on toolbox contents.
+ * @param {!Blockly.tree.BaseNode} treeOut The TreeControl or TreeNode
```

Comment:
I don't understand the descriptions of treeOut and the return value. Both are nodes--are they the *root nodes* of the things being described?